Common.mk

```diff
@@ -112,6 +112,7 @@ BUILDER_IMAGE?=$(BASE_IMAGE_REPO)/$(BASE_IMAGE_NAME)-builder:$(BASE_IMAGE_TAG)
 ####################################################
 
 #################### IMAGES ########################
+HAS_HELM_CHART?=false
```

Comment:
How about just moving this down to the HELM block?

Reply:
This gets used in the targets before the helm chart. I guess I could put the helm block before that. It was that way for a bit.

Reply:
It shouldn't matter where it is in the file vs other places its used since just about everything in here is ?= so its delayed evaluation anyway.